src/Files.App/Views/Pages/Properties.xaml.cs

```diff
@@ -286,4 +358,20 @@ public class PropertyNavParam
 			public IShellPage AppInstanceArgument { get; set; }
 		}
 	}
-}
\ No newline at end of file
+
+	public class SquareNavViewItem : ObservableObject
+	{
+		public string Name;
+
+		public string Tag;
```

Comment:
Let's use an enum instead of tag.

Reply:
I don't get what do you mean. An enum for this?

Reply:
Instead of tracking which item was clicked based on the tag, we should have an enum.